### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by sharped-rsc** [epam/ketcher#468] [General]
Hi,
I was just wondering if there is any indication of when the next version of Ketcher will be released. Is there a target date or feature set that is being worked towards?

Kind regards,

Dave

**Comment by AndreiMazol** [comment]
Hi @sharped-rsc ,

the next version of Ketcher will be released by the end of the week.
Key features of release 2.2:
- support v3000: now you  can save and open your mol v3000 files
- support png/svg generation from the structure: Ketcher API has been extended
- support of standalone mode: if you  want to work independently you can install and configure ketcher-standalone package that allows not to deploy Indigo Service (solution is based on WASM)
- CSS has been rewritten: now the size of Ketcher editor depends on parent node size that allows to render it not full screen

The whole list of implemented features and fixed bugs will be included in release notes.

Best regards,
Ketcher team

**Reply by sharped-rsc** [reply]
Thanks @AndreiMazol

**Reply by AndreiMazol** [reply]
@sharped-rsc ,

new version of Ketcher is available now.
Please look at [release notes](https://github.com/epam/ketcher/releases/tag/v2.2.0) for more details.
